### DISCUSSION TITLE
OV5640-AF Autofocus

### DISCUSSION THREAD

**Discussion by timosust** [jomjol/AI-on-the-edge-device#1753] [Ideas]
Moin,

bin gerade beim schnellen Ali am Stöbern, und sehe es gibt Kameras für den ESP32-Cam mit Autofocus.

[https://de.aliexpress.com/item/1005004072836395.html?spm=a2g0o.productlist.0.0.575931e6pMHJi2&algo_pvid=bc9fe6a2-019d-4dfe-98b6-667386f9add4&aem_p4p_detail=202301030046091437369141369200000171266&algo_exp_id=bc9fe6a2-019d-4dfe-98b6-667386f9add4-7&pdp_ext_f=%7B%22sku_id%22%3A%2212000027947676691%22%7D&pdp_npi=2%40dis%21EUR%2121.24%2116.35%21%21%21%21%21%4021038ed816727355691068397e30a7%2112000027947676691%21sea&curPageLogUid=EtApgwF6fZlg&ad_pvid=202301030046091437369141369200000171266_8&ad_pvid=202301030046091437369141369200000171266_8](url)

[https://www.arducam.com/products/camera-breakout-board/5mp-ov5640/](url)

[https://www.youtube.com/watch?v=922BWy3OOoQ](url)

Da die Suche hier nichts ergab, wollte ich das einfach mal in den Raum werfen, ob das bekannt ist.

Da ich keine Ahnung habe, wie aufwendig u schwer das zu implementieren wäre, explizit nicht als Request. Wobei es sicher schön wäre, wenn die Focusiererei wegfallen würde. Ist aber natürlich auch ein Kostenfaktor

MFG

**Comment by jomjol** [comment]
Muss man sich genauer anschauen, vermutlich funktionieren die Treiber nicht auf Anhieb. Eigentlich lohnt es sich nicht wirklich, da das Fokusieren nur ein Einmalaufwand ist.

**Reply by timosust** [reply]
Richtig, deswegen auch nur in die offene Runde geworfen. Wusste bis eben nicht, das es sowas gibt
